conf/app.houseConsignment.index.routes

```diff
@@ -14,6 +14,12 @@ POST       /:arrivalId/house-consignment/:houseConsignmentIndex/add-ucr
 GET        /:arrivalId/change-house-consignment/:houseConsignmentIndex/add-ucr                                     controllers.houseConsignment.index.UniqueConsignmentReferenceYesNoController.onPageLoad(arrivalId: ArrivalId, houseConsignmentIndex: Index, mode: Mode = CheckMode)
 POST       /:arrivalId/change-house-consignment/:houseConsignmentIndex/add-ucr                                     controllers.houseConsignment.index.UniqueConsignmentReferenceYesNoController.onSubmit(arrivalId: ArrivalId, houseConsignmentIndex: Index, mode: Mode = CheckMode)
 
+GET        /:arrivalId/house-consignment/:houseConsignmentIndex/ucr                                                controllers.houseConsignment.index.UniqueConsignmentReferenceController.onPageLoad(arrivalId: ArrivalId, houseConsignmentIndex: Index, mode: Mode = NormalMode)
+POST       /:arrivalId/house-consignment/:houseConsignmentIndex/ucr                                                controllers.houseConsignment.index.UniqueConsignmentReferenceController.onSubmit(arrivalId: ArrivalId, houseConsignmentIndex: Index, mode: Mode = NormalMode)
+GET        /:arrivalId/change-house-consignment/:houseConsignmentIndex/change-ucr                                  controllers.houseConsignment.index.UniqueConsignmentReferenceController.onPageLoad(arrivalId: ArrivalId, houseConsignmentIndex: Index, mode: Mode = CheckMode)
+POST       /:arrivalId/change-house-consignment/:houseConsignmentIndex/change-ucr                                  controllers.houseConsignment.index.UniqueConsignmentReferenceController.onSubmit(arrivalId: ArrivalId, houseConsignmentIndex: Index, mode: Mode = CheckMode)
```

Comment:
Think these two can still be `/ucr` for consistency since we have `/change-house-consignment`